pkg/model/dev.go

```diff
@@ -18,44 +18,47 @@ const (
 	// CNDDeploymentAnnotation is the original deployment manifest
 	CNDDeploymentAnnotation = "cnd.okteto.com/deployment"
 
-	// CNDDevAnnotation is the active cnd configuration
-	CNDDevAnnotation = "cnd.okteto.com/dev"
+	// CNDDevAnnotationPrefix is the prefix for cnd manifest annotations
+	CNDDevAnnotationPrefix = "cnd.okteto.com/dev-"
```

Comment:
I am using a new dev annotation. It is not a breaking change because it would only break multiple "cnd up"s to the same deployment, which was not possible before.